.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
- [ ] Great job setting up eslint :clap: , however please see that currently no eslint checks are running on github action workflows, this could be because you forgot to update the linters.yml file inside your workflows directory, let's update our linter.yml file to include the eslint jobs and also make sure that the eslint check is also running, you can get the [updated file from here ](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) this will make sure our javascript linters are setup correctly and the code we push to our remote repo is clean and follows best practices.

![image](https://user-images.githubusercontent.com/97521557/208486482-46a79929-787f-4595-8df3-be6dfbb9d40b.png)

Reply:
Thank you @PinkMoon25 , i will make the changes and request a new review